packages/site/src/components/NoMatch.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+
+const styles = {
+  container: {
+    textAlign: 'center'
+  },
+  error: {
+    fontSize: '10em',
+    fontWeight: 'bold',
+    margin: '1em 0 20px'
+  }
+};
+
+const Error = createStyledComponent('p', styles.error);
+const Container = createStyledComponent('div', styles.container);
+
+export default function NoMatch() {
```

Comment:
Perhaps `NotFound` might be more clean than `NoMatch`.   More importantly, why is this needed and how does it work, if 404's are supposed to redirect to the home page?